mail_delivery_robot/control/robotDriver.py

```diff
@@ -175,6 +184,12 @@ def updateDistance(self, data):
 
```

Comment:
There is no call to update the state here, so whats currently happening is only timer based instead of ROS interrupt based. Should this be how it works as it looks like it might not work too well.

Reply:
^ Line 175/184

Reply:
You're right, that will be added soon in the next pull.